### PR TITLE
RM - Fix the errors on surveys, guide and translations

### PR DESCRIPTION
**Pull request by Diama1**
#1191
#1193 
#1205

### CODE REVIEW

**Comment by adamhstewart** [comment]
@Diama1 can you provide screenshots to show the changes?

**Comment by adamhstewart** [comment]
@Diama1 Also, pretty sure this should go against https://github.com/TIP-Global-Health/eheza-app/pull/1159 before it is merged to the develop branch.

**Review by AronNovak — APPROVED** [review]

**Comment by Diama1** [comment]
> @Diama1 Also, pretty sure this should go against #1159 before it is merged to the develop branch.

@adamhstewart develop branch has the latest changes, the one I worked on that's why I branch out there. the other one is not yet merged and doesn't have the latest changes

**Comment by Diama1** [comment]
> @Diama1 can you provide screenshots to show the changes?

Screenshot from 2024-06-10 12-10-05
onload
whole

File: client/src/elm/Translate.elm [hunk 1]
```diff
@@ -18384,7 +18384,7 @@ translationSet trans =
 
                 ResilienceCategoryMindfulness ->
                     { english = "Mindfulness"
-                    , kinyarwanda = Just "Zirikana ibi"
+                    , kinyarwanda = Nothing
```
Comment (AronNovak) [inline client/src/elm/Translate.elm:18387]: Why do we remove a translation? Is it wrong? Can't we have the good one?

File: client/src/elm/Translate.elm [hunk 2]
```diff
@@ -18762,19 +18762,19 @@ translationSet trans =
...
             { english = "Self-care"
-            , kinyarwanda = Just "Kunanirwa"
+            , kinyarwanda = Just "Kwiyitaho"
             , kirundi = Nothing
             }
 
         ResilienceGuideSection3Bullet7 ->
             { english = "Mindfulness"
-            , kinyarwanda = Just "Zirikana ibi"
+            , kinyarwanda = Nothing
```
Comment (AronNovak) [inline client/src/elm/Translate.elm:18777]: same here

File: client/src/elm/Pages/MessagingCenter/View.elm [hunk 3]
```diff
@@ -66,74 +66,76 @@ view language currentTime nurseId nurse db model =
...
+                                if runQuarterlySurvey then
+                                    viewQuarterlySurvey language currentDate nurseId model.surveyForm
+
+                                else if runAdoptionSurvey then
+                                    viewAdoptionSurvey language currentDate nurseId model.surveyForm
+
+                                else
+                                    messagingCenterView
+                            )
+                        |> Maybe.withDefault messagingCenterView
```
Comment (anvmn) [inline client/src/elm/Pages/MessagingCenter/View.elm:138]: @Diama1 This means that if nurse got no surveys filled, we will never ask to fill any survey.
So, this change is incorrect.

**Review by anvmn — CHANGES_REQUESTED** [review]

**Comment by anvmn** [comment]
Moved to #1159. Closing.

**Pull request closed without merging**